src/blend/src/Shared/Blend/Blend.lua

```diff
@@ -1,3 +1,4 @@
+---@diagnostic disable: redefined-local
```

Comment:
nit: We should not disable this redefined-local lint, and instead we should actually not redefine a local value.